src/app/components/home.component.ts

```diff
@@ -1695,13 +1698,13 @@ export class HomeComponent implements OnInit, AfterViewInit {
       clipSnippets: 5,
       extractClips: false,
       futureHubName: '',
-      isFixedNumberOfScreenshots: true,
-      screenshotSizeForImport: 288, // default
+      isFixedNumberOfScreenshots: this.wizard.isFixedNumberOfScreenshots ?? true,
```

Comment:
I'm loving the [nullish-coalescing](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) here 👍